slash_commands.js

```diff
@@ -70,3 +71,52 @@ changelog.handle = subCommandMatcher(changelog.slashCommand, {
     await checkChangelog({prNumber: `${prNumber}`});
   }
 });
+
+const depGraph = {
+    slashCommand: '/dep-graph'
+};
+const commentDepGraph = monoRepoWorkflowDispatch(octokit, 'comment-dependency-graph');
+const commentDepGraphHelpMessage = ({octokit, prLink}) => {
+    const prLinkRegex = /https:\/\/github.com\/hasura\/(.*)\/pull\/(\d+)/;
+    const [, repoName, prNumber] = prLink.match(prLinkRegex);
+
+    octokit.issues.createComment({
+        owner: 'hasura',
+        repo: repoName,
+        issue_number: prNumber,
+        body: depGraphHelpMessage
+    });
+}
+
+depGraph.check = slashCommandChecker(depGraph.slashCommand);
+depGraph.handle = subCommandMatcher(depGraph.slashCommand, {
+    'SERVER': async ({octokit, prLink}) => {
+        console.log('commenting dependency graph for server changes');
+        await commentDepGraph({
+            prLink
```

Comment:
should there be a `,` at the end? :thinking:

Reply:
ah yes :sweat_smile: